addon/services/popover.js

```diff
@@ -1,12 +1,99 @@
 import Ember from 'ember';
+import { assert } from '@ember/debug';
+import ModalComponent from 'ember-widgets/components/modal-component';
+import PopoverComponent from 'ember-widgets/components/popover-component';
+import $ from 'jquery';
+
+function classInheritsFrom(childKlass, parentKlass) {
+  return parentKlass.prototype.isPrototypeOf(childKlass.prototype);
+}
 
 export default Ember.Service.extend({
   init() {
     this._super(...arguments);
+
+    // Array of specifications for modals and/or popovers
     this.popovers = Ember.A([]);
   },
 
-  open(destinationElement, ComponentClass, options) {
+  openPopover(componentName, options={}, hideAll=true) {
+    assert(`[ember-widgets#openPopover] {{render-popover}} must be included in a template for popovers to open`, !!this._popoverDidRender);
+
+    let ComponentClass = this.container.lookupFactory(`component:${componentName}`);
```

Comment:
There are some views that get rendered via `.popover()`. Might need to pass in the whole container registry string, or at least an option to specify a view.

Reply:
It's very possible I got fooled by [some aliasing](https://github.com/Addepar/Iverson/blob/master/iverson/app/views/report-widget-popover-content.js#L5). I'm digging through now to sanity check if there are any view-based popovers.

Reply:
Very happy to say that's the only one  🎉

Reply:
@kybishop Thanks for finding that. I'm actually unclear on what that is doing. Tests seem to pass even though I haven't modified it. Do you have an idea for what to do about it?

@mixonic I pulled in the lookups for `ComponentClass` into the body of their respective assertions (in both `openModal` and `openPopover`), so those will now be stripped.

Reply:
Actually, I spoke to soon about moving `lookupFactory` — we need to look at the `ComponentClass` to figure out the modal|popover destination element. So I will pull back that reference. I previously looked in the app and do not see any/many places where there is `rootElement` on the `ComponentClass`, so we can probably remove that altogether soon, after taking another pass. But I'll leave it as is for now to land this PR.